### PR TITLE
Initialized components

### PR DESCRIPTION
**Pull request by thecodechaser**
## This branch includes the following features:
-  Created React app.
-  Added Redux.
-  Added components Books, Categories, Header.
-  Structured Redux files.
-  Applied styling.

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+const Books = () => (
+  <main>
+    <div />
+    <h2 className="add-book-h2">ADD NEW BOOK</h2>
+    <form className="form">
+      <input className="input-title" type="text" placeholder="Book title" />
+      <input className="input-category" type="text" placeholder="Category" />
+      <button className="add-book-btn" type="submit">Add Book</button>
+    </form>
```
Comment (TedLivist) [inline src/components/Books.js]: - [ ] In keeping with the SOLID principle kindly extract this form into a new separate component and render that component in this **`Books`** component instead.

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const Books = () => (
```
Comment (TedLivist) [inline src/components/Books.js:5]: - [ ] Kindly create a new component (eg `Book`) that represents a single book. This component should take in book data as a prop and render its title and author
- [ ] Kindly ensure that the new aforementioned component (eg `Book`) has a placeholder Remove button (this button does not have to work yet - it is merely a placeholder)
- [ ] Your **`Books`** component (which you already have) should render a list of the created `Book` components

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @thecodechaser,

Good job so far!
✔️ Components are rightly placed in the correct directory
✔️ React routes are well set up
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 1 above)
Comment (peterrobert) [reply src/components/Books.js]: This change has been done succesfully.

File: src/components/Books.js (hunk 2 above)
Comment (peterrobert) [reply src/components/Books.js:5]: The second and the third points have been done well. However, one request is not yet met. Kindly check below for more information.

Comment (thecodechaser) [reply src/components/Books.js:5]: > The second and the third points have been done well. However, one request is not yet met. Kindly check below for more information.

@peterrobert Hello

Comment (peterrobert) [reply src/components/Books.js:5]: All changes have been done well

**Review by peterrobert — APPROVED** [review]
Hi @thecodechaser, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by thecodechaser** [comment]
> Hi @thecodechaser,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@peterrobert Thank you brother 👍

**Pull request merged**